pyproject.toml

```diff
@@ -40,7 +40,7 @@ marshmallow = "==3.20.2"
 marshmallow-sqlalchemy = "==0.30.0"
 newrelic = "*"
 notifications-python-client = "==9.0.0"
-notifications-utils = {git = "https://github.com/GSA/notifications-utils.git"}
+notifications-utils = {git = "https://github.com/GSA/notifications-utils.git", rev="053fe30"}
```

Comment:
@terrazoon has the `rev` property been working for you when running `make bootstrap` to pull in newer updates of `notification_utils`?

That didn't seem to be working for us before but if it is now, this would eliminate the need to do a whole release and update as a separate step, which would be awesome!

Reply:
The only potential downside with that is we would need to modify every single time we made an update in utils that we want to pull in as opposed to one time with an update release that has everything merged.

Reply:
rev works for me, but it's a mistake for it to be in this PR.  I was just testing against my version of notifications-utils in my other PR.  I would remove it prior to merge (after merging other PR first).

Reply:
Okay that makes sense, thanks for confirming!